{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--a34bb581-6a50-447f-9a3d-c4764bfa9d15",
    "created": "2020-05-26T20:29:47.140538Z",
    "modified": "2020-05-26T20:29:47.140538Z",
    "name": "CVE-2018-14614",
    "description": "An issue was discovered in the Linux kernel through 4.17.10. There is an out-of-bounds access in __remove_dirty_segment() in fs/f2fs/segment.c when mounting an f2fs image."
}